packages/frontend/src/ee/features/embed/EmbedDashboard/components/EmbedDashboard.tsx

```diff
@@ -117,6 +215,56 @@ const EmbedDashboard: FC<{
             [dashboard],
         ) || false;
 
+    // Tab state
+    const [activeTab, setActiveTab] = useState<DashboardTab | undefined>();
+
+    // Sort tabs by order
+    const sortedTabs = useMemo(() => {
+        if (!dashboard?.tabs || dashboard.tabs.length === 0) {
+            return [];
+        }
+        return dashboard.tabs.sort((a, b) => a.order - b.order);
+    }, [dashboard?.tabs]);
+
+    // Set active tab to first tab if no active tab is set
+    useEffect(() => {
+        if (sortedTabs.length > 0 && !activeTab) {
+            setActiveTab(sortedTabs[0]);
+        }
+    }, [sortedTabs, activeTab]);
```

Comment:
Not a huge thing, but if we broke up the data fetching portion of the dashboard from the presentation part of the dashboard, we could get rid of the `useEffect`. In that case, activeTab could be set when that component's useState is initialized. 

Probably not worth it now at this point though. There are already a lot of `useEffects` in this component.